sdk/identity/Azure.Identity/src/AzureIdentityEventSource.cs

```diff
@@ -8,11 +8,12 @@
 using System.Text;
 using Azure.Core;
 using Azure.Core.Diagnostics;
+using Microsoft.IdentityModel.Abstractions;
 
 namespace Azure.Identity
 {
     [EventSource(Name = EventSourceName)]
-    internal sealed class AzureIdentityEventSource : AzureEventSource
+    internal sealed class AzureIdentityEventSource : AzureEventSource, IIdentityLogger
```

Comment:
Just for my understanding, what's the relationship between this class and AzureEventSourceListener?

Reply:
only that the calls to `IsEnabled` consult the attached listeners to the event source.